### PR TITLE
Remove testAreaContext from test_ecl_run

### PR DESCRIPTION
**Pull request by oyvindeide**
testAreaContext might be creating problems by keeping file handles open, so switching to tmpdir seems reasonable.

### CODE REVIEW

File: python/tests/res/fm/test_ecl_run.py [hunk 1]
```diff
@@ -94,84 +94,80 @@ def init_flow_config(self):
...
-            ta.copy_file( os.path.join(self.SOURCE_ROOT , "test-data/local/eclipse/SPE1.DATA"))
-            ta.copy_file( os.path.join(self.SOURCE_ROOT , "test-data/local/eclipse/SPE1_ERROR.DATA"))
-            os.makedirs("ecl_run")
-            shutil.move("SPE1.DATA", "ecl_run")
-            shutil.move("SPE1_ERROR.DATA" , "ecl_run")
-            flow_config = FlowConfig()
-            sim = flow_config.sim()
-            flow_run = EclRun("ecl_run/SPE1.DATA", sim)
-            flow_run.runEclipse( )
+        self.init_flow_config()
```
Comment (andreabrambilla) [inline python/tests/res/fm/test_ecl_run.py:160]: I suppose this test should get the `use_tmpdir` fixture as well, right?

**Comment by andreabrambilla** [comment]
once this works, can we also remove the `from ecl.util.test import TestAreaContext` on line 27, or do we still need it in some cases?

File: python/tests/res/fm/test_ecl_run.py (hunk 1 above)
Comment (oyvindeide) [reply python/tests/res/fm/test_ecl_run.py:160]: Yes, which is in #727, though we might consider rewriting the whole tmpdir implementation somewhat based on feedback in that PR.

Comment (andreabrambilla) [reply python/tests/res/fm/test_ecl_run.py:160]: I am not sure I follow, in #727 `test_ecl_run.py` is not touched

**Comment by oyvindeide** [comment]
> once this works, can we also remove the `from ecl.util.test import TestAreaContext` on line 27, or do we still need it in some cases?

Good catch, yes, have updated

File: python/tests/res/fm/test_ecl_run.py (hunk 1 above)
Comment (oyvindeide) [reply python/tests/res/fm/test_ecl_run.py:160]: You're right, I misread your comment, I've removed TestAreaContext from test_flow

**Review by andreabrambilla — APPROVED** [review]

**Pull request merged**